compat/compat.go

```diff
@@ -0,0 +1,66 @@
+package compat
+
+import (
+	chaincfg2 "github.com/decred/dcrd/chaincfg/v2"
+	chaincfg3 "github.com/decred/dcrd/chaincfg/v3"
+	dcrutil2 "github.com/decred/dcrd/dcrutil/v2"
+	dcrutil3 "github.com/decred/dcrd/dcrutil/v3"
+	hdkeychain2 "github.com/decred/dcrd/hdkeychain/v2"
+	hdkeychain3 "github.com/decred/dcrd/hdkeychain/v3"
+)
+
+func Params2to3(v2 *chaincfg2.Params) *chaincfg3.Params {
+	// Crappy way to do this, but dcrlnd doesn't change any params anywhere
+	// (and this is temporary code) so whatever.
```

Comment:
If it is temporary, you can just comment with `//ToDo: temporary because dcrlnd doesn'tt change any params anywhere`
the same way from another comment.

Reply:
The entire `compat` module will go away once all packages are updated.